multipost.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+     <meta charset="UTF-8" >
+     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+     <link rel="stylesheet" href="multipost.css">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+     <title>Mobile Version Website</title>
+  </head>
+    <body>
+     <div class="contmultipost">
+<section class="post">
```

Comment:
Kindly note that you should only have one html file and one css file. Currently you have the works section as a different page. It should make up a section in the page you did the headline in.
The structure of your page should be
`index.html`
`<html>`
`<body>`
`</header><nav>... </nav> </header>`

`<section>headline <section>`

`<section>works<section>`

`<section>About <section>`

`</body>`
`</html>`

Refer back to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)
<img width="1666" alt="Screenshot 2022-10-29 at 10 38 24" src="https://user-images.githubusercontent.com/48546992/198820100-3787b680-a628-4041-8757-62de2b7c7423.png">